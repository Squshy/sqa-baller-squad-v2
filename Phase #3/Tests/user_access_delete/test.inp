LOGIN
BABYYODA
lilcuteguy
DELETE
logout
exit