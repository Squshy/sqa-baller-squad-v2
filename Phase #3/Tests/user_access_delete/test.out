=========================================================
             BALLER SQUAD'S AUCTION EMPORIUM!
=========================================================

                What would you like to do?

        Bid              Auction            Refund

        Update           Delete             Logout               


                         DELETE

---------------------------------------------------------
      You must be an admin to access this transaction.
=========================================================